source/features/separate-draft-pr-button.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+
+function init(): void {
+	const previewForm = select<HTMLFormElement>('.new-pr-form');
+
+	// PRs can't be created from some comparison pages:
+	// Either base is a tag, not a branch; or there already exists a PR.
+	if (!previewForm) {
+		return;
+	}
+
+	const buttonBar = select('.timeline-comment > :last-child', previewForm)!;
+	const createPrButtonGroup = select('.BtnGroup', buttonBar)!;
+	const createPrDropdownItems = select.all('.select-menu-item', createPrButtonGroup);
+
+	for (const dropdownItem of createPrDropdownItems) {
+		let title = select('.select-menu-item-heading', dropdownItem)!.textContent!.trim();
```

Comment:
I think you can skip the `select` altogether and just set the `title` in the following condition block

Reply:
I can't. The regex in the condition is tested against this title.

Reply:
D’oh